### DISCUSSION TITLE
If no streaming data comes in, will the rtsp-simple service process exit by itself?

### DISCUSSION THREAD

**Discussion by DreamyCheng** [bluenviron/mediamtx#1017] [Questions]
If no streaming data comes in, will the rtsp-simple service process exit by itself?
If exit, how long?

**Comment by aler9** [comment]
Hello, the server works exactly like any other server, it stays open independently from the fact that there are connected clients or not.

**Reply by DreamyCheng** [reply]
It appeared twice early on, but it seemed to be fine after restarting the server. The service has now been running normally for 29 days.
This problem is not repeated now, it may be a problem of the machine

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
root@localhost ~]# ps aux | grep rtsp-simple-server
root       64409  0.3  0.0 728512 38080 ?        Sl   Jul11 138:52 ./rtsp-simple-server
root     2212797  0.0  0.0   7424  1600 pts/1    S+   12:18   0:00 grep rtsp-simple-server
[root@localhost ~]# ps -eo pid,tty,user,comm,lstart,etime|grep "rtsp"
  64409 ?        root     rtsp-simple-ser Mon Jul 11 12:17:58 2022 29-00:00:58

[a comment by an automated account is not shown]
